qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst

```diff
@@ -71,17 +71,22 @@ References:
 Shotgun sequencing
 ------------------
 
-Qiita currently has one active shotgun metagenomics data analysis pipeline: a per sample
+Qiita currently has one active shotgun metagenomics data analysis pipeline: a per sample, paired-end
 bowtie2 alignment step with Woltka classification using either the WoLr2 (default) or RS210 databases.
 Below you will find more information about each of these options.
 
 .. note::
-   The bowtie2 settings are maximum and minimum mismatch penalties (mp=[1,1]), a
-   penalty for ambiguities (np=1; default), read and reference gap open- and
+   The bowtie2 settings are set for interleaved processing with a maximum and minimum mismatch
+   penalties (mp=[1,1]), a penalty for ambiguities (np=1; default), read and reference gap open- and
```

Comment:
"reference gap open- and" seems odd. Perhaps it should be "open-and"? or maybe the hyphen should be removed?